pkg/util/util.go

```diff
@@ -129,3 +132,33 @@ func UnArchiveLocalTar(filePath, destDir string, arg ...string) error {
 	fileReader := bufio.NewReader(file)
 	return UnArchiveTar(fileReader, destDir, arg...)
 }
+
+// CopyFile copies a file from one location to another.
+func CopyFile(src, dst string) error {
+	in, err := os.Open(src)
+	if err != nil {
+		return err
+	}
+	defer in.Close()
+
+	out, err := os.Create(dst)
+	if err != nil {
+		return err
+	}
+	defer out.Close()
+
+	_, err = io.Copy(out, in)
+	if err != nil {
+		return err
+	}
+	return out.Close()
```

Comment:
Is it ok that Close() is called twice for out (return statement and defer)?

Reply:
Yes you can call close multiple times on files. In general though the behaviour is undefined calling close twice on a Closer. For files it works fine.